feat(framework): Update changelog for 1.32.0 release (#7473)
adap/flower: framework/docs/source/ref-changelog.md

### What's new?

- **Introduce experimental agent features in SuperGrid** ([#7336](https://github.com/flwrlabs/flower/pull/7336), [#7341](https://github.com/flwrlabs/flower/pull/7341), [#7344](https://github.com/flwrlabs/flower/pull/7344), [#7345](https://github.com/flwrlabs/flower/pull/7345), [#7346](https://github.com/flwrlabs/flower/pull/7346), [#7351](https://github.com/flwrlabs/flower/pull/7351), [#7360](https://github.com/flwrlabs/flower/pull/7360), [#7366](https://github.com/flwrlabs/flower/pull/7366), [#7379](https://github.com/flwrlabs/flower/pull/7379), [#7397](https://github.com/flwrlabs/flower/pull/7397), [#7409](https://github.com/flwrlabs/flower/pull/7409), [#7418](https://github.com/flwrlabs/flower/pull/7418), [#7419](https://github.com/flwrlabs/flower/pull/7419), [#7435](https://github.com/flwrlabs/flower/pull/7435), [#7440](https://github.com/flwrlabs/flower/pull/7440), [#7441](https://github.com/flwrlabs/flower/pull/7441), [#7468](https://github.com/flwrlabs/flower/pull/7468))

>   Introduces experimental agent capabilities in SuperGrid through `AgentApp` and connectors. Adds web search connectors with Brave, Tavily, and Exa providers, a web fetch connector, and a browser-use connector, while enabling connector-based task execution and integration with `AgentApp` workflows.

Author: @danielnugraha Wdyt?

Reviewer: What about simplifying it to

Introduce experimental agent capabilities in SuperGrid with AgentApp connectors for web search, web fetch, and browser use.

Author: ```suggestion
  Introduces experimental agent capabilities in SuperGrid with `AgentApp` connectors for web search, web fetch, and browser use.
```

# Changelog

## v1.32.0 (2026-06-25)

### Thanks to our contributors

We would like to give our special thanks to all the contributors who made the new version of Flower possible (in `git shortlog` order):

`Aleksei Balan`, `Charles Beauville`, `Chong Shen Ng`, `Daniel J. Beutel`, `Daniel Nata Nugraha`, `Heng Pan`, `Javier`, `Julian Rußmeyer`, `Micah Sheller`, `Mohammad Naseri`, `Patrick Foley`, `Ruth Galindo`, `William Lindskog`, `reducedradius` <!---TOKEN_v1.32.0-->

### What's new?

- **Introduce default federations on SuperGrid** ([#7436](https://github.com/flwrlabs/flower/pull/7436), [#7437](https://github.com/flwrlabs/flower/pull/7437), [#7460](https://github.com/flwrlabs/flower/pull/7460), [#7486](https://github.com/flwrlabs/flower/pull/7486), [#7488](https://github.com/flwrlabs/flower/pull/7488), [#7491](https://github.com/flwrlabs/flower/pull/7491), [#7492](https://github.com/flwrlabs/flower/pull/7492))

>   Introduces default federations for every Flower account in SuperGrid, allowing `flwr run` commands targeting SuperGrid to run without explicitly specifying `--federation`.

Reviewer 2: ```suggestion
  Introduces default federations for every Flower account in SuperGrid, allowing `flwr run` commands targeting SuperGrid to run without explicitly specifying `--federation`. Run your first Flower App on SuperGrid by taking [the tutorial](https://flower.ai/docs/framework/1.32/en/tutorial-series-get-started-with-flower.html). 
```
